src/tools/tidy/src/main.rs

```diff
@@ -19,6 +19,11 @@ use std::path::{PathBuf, Path};
 use std::env;
 
 macro_rules! t {
+    ($e:expr, $p:expr) => (match $e {
+        Ok(e) => e,
+        Err(e) => panic!("{} failed on {} with {}", stringify!($e), ($p).display(), e),
+    });
```

Comment:
Right yeah this is fine to leave for now, eventually we could perhaps add some wrappers around calling these functions in bootstrap (which add the path as a payload to the error) and then change this later.